### DISCUSSION TITLE
Feature Request: CSS Syntax highlighting

### DISCUSSION THREAD

**Discussion by apetta** [pieces-app/support#168] [Ideas]
CSS snippets shown as grey text, would be good to have syntax highlighting like code editors

**Comment by caleb-at-pieces** [comment]
Hello @apetta thank you for pointing this out to us. Is there a specific application that css is showing up as grey text in?

**Reply by apetta** [reply]
Hey! Seems to be just the Pieces Mac app and the Chrome extension (Mac) where there's no highlighting. It's working great in the VS Code extension.

![Screenshot 2024-04-16 at 14 17 58](https://github.com/pieces-app/support/assets/33671231/97e1ff2f-2534-4621-9b3a-c92066d08ef6)

![Screenshot 2024-04-16 at 14 18 16](https://github.com/pieces-app/support/assets/33671231/7fa2fa8b-5a7c-497a-9247-bc3518a585da)

**Reply by apetta** [reply]
VS Code screenshot:
![Screenshot 2024-04-16 at 14 20 46](https://github.com/pieces-app/support/assets/33671231/98c58060-9850-449a-b142-f319df652667)
